var/spack/repos/builtin/packages/ghostscript-fonts/package.py

```diff
@@ -8,11 +8,12 @@
 from spack.package import *
 
 
-class GhostscriptFonts(Package):
+class GhostscriptFonts(Package, SourceforgePackage):
     """Ghostscript Fonts"""
 
     homepage = "https://ghostscript.com/"
-    url = "https://www.imagemagick.org/download/delegates/ghostscript-fonts-std-8.11.tar.gz"
+    sourceforge_mirror_path = "gs-fonts/gs-fonts/8.11%20%28base%2035%2C%20GPL%29/" + 
+                              "ghostscript-fonts-std-8.11.tar.gz"
```

Comment:
This is invalid syntax. I think people use more often parenthesis, but if not you need `\` after the `+` sign to have a multiline expression.

Reply:
Alright, I should definitely have tested that...

Reply:
🤦